pkg/cache/resource.go

```diff
@@ -44,3 +46,15 @@ func remainingQuota(node netQuotaNode) resources.FlavorResourceQuantities {
 	}
 	return remainingQuota
 }
+
+type lendableNode interface {
+	getResourceNode() ResourceNode
+}
+
+func calculateLendable(node lendableNode) map[corev1.ResourceName]int64 {
```

Comment:
maybe this could be a function of `FlavorResourceQuantities` to merge lendables for all flavors of a resource.

Reply:
It feels a little easier to use when we  define this as a function of the node, as the caller won't have to dig inside to know that it is the SubtreeQuota.